agent/05_agent_create_cluster.sh

```diff
@@ -54,7 +54,7 @@ function wait_for_cluster_ready() {
   sleep 5m
```

Comment:
I think that also this block could be removed, given that we don't have to wait anymore for the kubeconfig availabiity

Reply:
I haven't tried but if we start oc before the bootstrap control plane is up, won't it fail?

Reply:
Yes, you're right, we could keep the above block as a poor-man wait-for until the installer command will be ready